oauth2client/flask_util.py

```diff
@@ -191,6 +192,21 @@ def requires_calendar():
 __author__ = 'jonwayne@google.com (Jon Wayne Parrott)'
 
 DEFAULT_SCOPES = ('email',)
+CREDENTIALS_KEY = 'google_oauth2_credentials'
```

Comment:
How about leading underscores? Or do you intend for these values to be used outside this module?